### DISCUSSION TITLE
Additive health rating

### DISCUSSION THREAD

**Discussion by vinniec** [SuhasDissa/Food-E-App#63] [Ideas]
a useful app, when I saw it it reminded me of this other very similar app:
https://f-droid.org/packages/org.uaraven.e/

the difference with your application is that with colors and icons it immediately shows if an additive is considered dangerous.
Your application  however explains what each individual additive is, which I find very interesting.

Maybe you could complement the two, what do you think?

**Comment by SuhasDissa** [comment]
Hi there, thank you for your comment.

I did try to add such health rating in the past. However I couldn't find any reliable data sources.

I could've just copied the ratings from this app. But since I'm not an additive expert, I wasn't sure if those information is accurate and up-to-date.
So, I thought of not adding it anyway.
Having false information can cause more damage than not having such information. 

Hope you understand.
Thank you again.

**Comment by vinniec** [comment]
Thanks for your Kindly response,
I can understand, yours is a shareable position.

I can only say that it would be nice if in this world even the ill-intentioned people had the same scruples as the well-intentioned ones :p
As if to say that those who want to do well often don't do it for fear of making mistakes, while those who want to do badly are never afraid of being wrong!

**Comment by inson1** [comment]
In our country we have non profit app with products health ratings, Es ratings (they created it themselves), but its not open source and has ads :/
Its in czech, but Google translate can help.

--- https://www.ferpotravina.cz/ecka

![brave_j6sXrPA1p3](https://github.com/SuhasDissa/Food-E-App/assets/75314629/f826bfc0-2068-497e-a88b-7ef0a3e91724)

--- https://www.ferpotravina.cz/seznam-ecek

- https://www.ferpotravina.cz/maslo/jihoceske-maslo-madeta-a-s (butter)
- https://www.ferpotravina.cz/termizovany-krem/lipanek-vanilkovy-madeta-a-s (yogurt)
- https://www.ferpotravina.cz/limonada/coca-cola-coca-cola-hbc-cr-s-r-o
- https://www.ferpotravina.cz/bonbony/bonbony-s-prichuti-jablko-mata (candy)

![brave_IIFb2qWo5P](https://github.com/SuhasDissa/Food-E-App/assets/75314629/d5064de2-01e7-4338-8aee-cc887ffc0492)

**Comment by inson1** [comment]
And here are 3 ratings - safe, controversial, dangerous
https://nutrelino.com/en/additives/list/1
